src/terminal.rs

```diff
@@ -1,55 +1,71 @@
-#[cfg(feature="v0_48")]
 use std::path::PathBuf;
 use std::ptr;
 
 use ffi;
 use gdk;
 use glib::translate::*;
-#[cfg(feature="v0_48")]
```

Comment:
Yes, I meant these changes.
It's fine then.
I'll try to take a look at you pull request this weekend.